aostypes/aostypes.go

```diff
@@ -95,6 +95,14 @@ type BoardConfig struct {
 	Resources     []ResourceInfo `json:"resources"`
 }
 
+// Monitoring configuration for system monitoring.
+type Monitoring struct {
```

Comment:
you could move this structure to monitorgin
type Config struct

Reply:
Yes, let's move it to monitoring package as monitoring.Config.

Reply:
Done